src/client/src/components/NavBar.vue

```diff
@@ -1,29 +1,56 @@
 <template>
   <v-app-bar color="white" app dark>
-
     <div @click="base" class="d-flex align-center">
-      <v-img class="shrink mr-2" contain :src="logo" 
-      transition="scale-transition" width="200"/>
+      <a>
+        <v-img
+          class="shrink mr-2"
+          contain
+          :src="logo"
+          transition="scale-transition"
+          width="200"
+        />
+      </a>
     </div>
-    <v-btn @click="home" class="ml-4" color="black" text>
-      <h2 class="text-capitalize">Home</h2>
-    </v-btn>
-    <v-btn color="black" text>
-      <h2 class="text-capitalize">About</h2>
-    </v-btn>
 
+    <v-container class="pa-2" align-end justify-end>
+      <v-col>
+        <v-toolbar
+          class=".rounded-xl"
+          dense
+          floating
+          rounded
+          outlined="true"
+          light
+        >
+          <v-text-field
+            hide-details
+            color="black"
+            placeholder="Search"
```

Comment:
The searchbar looks really unprofessional, maybe remove the shadow and make the text input box slim

Reply:
i'm currently making an edit rn. check code-reviews channel for screenshot